WEB/Src/PerformanceCollector/PerformanceCollector/Perf.csproj

```diff
@@ -30,6 +30,17 @@
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
     <PackageReference Include="Microsoft.Extensions.Caching.Memory" Version="1.0.0" />
     <PackageReference Include="System.Diagnostics.PerformanceCounter" Version="4.7.0" />
+    
+    <!--
+    System.Drawing.Common 4.7.0 has a vulnerability https://msrc.microsoft.com/update-guide/en-US/vulnerability/CVE-2021-24112
```

Comment:
System.Diagnostics.PerformanceCounter --> Is it possible to pick a new version of this package instead?

Reply:
Yes, see my alternatives considered.

Reply:
I think it's very odd for our sdk to have a "Drawing" dependency.

If we upgrade System.Diagnostics.PerformanceCounter, we would have to jump two major versions to get a version of `System.Drawing.Common` without the vulnerability. I already confirmed that this supports NetStandard2.0 which is what we need.



v4.7.0
![image](https://user-images.githubusercontent.com/28785781/200052242-861c4bf6-d30f-4c18-84eb-dd41c9e9c21f.png)


v6.0.0
![image](https://user-images.githubusercontent.com/28785781/200052411-95a81c62-bb74-4a33-b8de-5e0fa3acd895.png)